packages/kit/test/apps/basics/src/routes/redirect-on-load/_tests.js

```diff
@@ -0,0 +1,13 @@
+import * as assert from 'uvu/assert';
+
+/** @type {import('test').TestMaker} */
+export default function (test, is_dev) {
```

Comment:
`is_dev` is unused

```suggestion
export default function (test) {
```

Reply:
removed!

BTW, Some existing tests also declared unused `is_dev`.
But test files have been excluded from `pnpm lint` and `pnpm check`.
So it has a chance to have a mistake like this.

Reply:
linting is enabled now: https://github.com/sveltejs/kit/commit/a2c9a15ec5604a5216fa3940768421bf838fa71c

it's not catching unused variables in other tests though, so perhaps there's some other check that would need to be enabled for that

Reply:
I think because `.eslintrc.json` extends `@sveltejs` but `no-unused-vars` is off.

Should I add it? (If yes I will create one more PR.)
```
"rules": {
    "no-unused-vars": "error"
}
```

https://github.com/sveltejs/kit/blob/a2c9a15ec5604a5216fa3940768421bf838fa71c/packages/kit/.eslintrc.json#L3

[sveltejs/eslint-config]
https://github.com/sveltejs/eslint-config/blob/31fd4faeea88990069502460b023698b1c9c2d13/index.js#L15

Reply:
We don't use that rule because it's the JavaScript rule and won't work on TypeScript files. Instead we've turned on the TypeScript version of the rule: https://github.com/sveltejs/eslint-config/blob/31fd4faeea88990069502460b023698b1c9c2d13/index.js#L48

It looks like that rule is essentially disabled because it is set to `none`. It would need to be changed to something like `after-used` to enable it.

We try to keep the rules for this repo in sync with the `sveltejs/svelte` repo, so any changes would need to happen in the following order:
- Enable the check locally in `sveltejs/svelte` and fix all errors, but don't check in the change to `.eslintrc.json`
- Modify the `sveltejs/eslint-config` package to update the rule after the PR to `sveltejs/svelte` is merged so that we know people agree to the code changes resulting from changing the eslint config value
- Update `sveltejs/svelte` and `sveltejs/kit` with the new version of `sveltejs/eslint-config` and fix any remaining errors

Reply:
Thank you for tell me how to change lint settings for svelte.

> It looks like that rule is essentially disabled because it is set to none. It would need to be changed to something like after-used to enable it.

I enabled below rule locally in `sveltejs/svelte`.
But 246 errors come. lol

```
rules: {
  '@typescript-eslint/no-unused-vars': ['error', { args: 'after-used' }]
}
```

Of cause I can fix these errors.
But I'm afraid that it will have a significant impact on other projects.
(I can look around at all the projects but should we?)

So Is there a way to get prior approval from the core team?
Or can we consider this chat as an approval?

Reply:
No need to worry about other projects. But the change would need approval from the maintainers besides myself. You could possibly try sending a PR fixing a good handful of the errors for discussion so that people can see what the changes would be without you having to do the work of fixing all of them.

Reply:
Okay! I will try to send a PR.
Thank you for supporting me!